app/controllers/concerns/foreman/controller/environments.rb

```diff
@@ -21,7 +21,13 @@ def import_environments
     if @changed["new"].size > 0 || @changed["obsolete"].size > 0 || @changed["updated"].size > 0
       render "common/_puppetclasses_or_envs_changed"
     else
-      notice _("No changes to your environments detected")
+      notice_message = _("No changes to your environments detected")
+
+      if @changed['ignored'].present?
+        notice_message << "\n" + _("Ignored environments: %s") % @changed['ignored'].keys.to_sentence
```

Comment:
The message here is including names of environments that contain _some_ ignored classes (via the filter), not only environments that are entirely ignored, which is misleading.